clients/status.h

```diff
@@ -45,6 +45,7 @@ static struct {
 	{ "BOOST",	"VOLTAGE BOOST",	1	},
 	{ "CAL",	"CALIBRATION",		1	},
 	{ "BYPASS",	"BYPASS",		2	},
+	{ "ECO",	"ECO",			2	},
```

Comment:
Severity 2 seems a bit excessive here (putting `ECO` in the same category as `LB`) - Severity 1 instead?